_tips/writing.id.md

```diff
@@ -0,0 +1,470 @@
+* **Tutorial**
+  * [Gambar](/tutorials/images/)
+* **User Story**
+  * [Menjelaskan nilai dari alternatif teks bagi pengguna penyandang disabilitas netra](/people-use-web/user-stories/#accountant)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Buat transkripsi dan takarir untuk multimedia
+
+Untuk konten audio, seperti podcast, berikan transkripsinya. Untuk konten audiovisual, seperti video pelatihan, sediakan pula takarirnya. Cantumkan pada transkripsi dan takarir informasi terkait ucapan lisan dan suara yang penting diketahui untuk memahami konten, misalnya, 'pintu berderit'. Untuk transkripsi video, sertakan juga deskripsi konten visual yang penting, misalnya 'Athan meninggalkan ruangan'.
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Informasi Lebih Lanjut" class="simple" %}
+{:/}
+
+* **[[Membuat Media Audio dan Video yang Aksesibel]](/media/av/)**
+
+* **WCAG**
+  * [Takarir (Prarekaman) 1.2.2](/WAI/WCAG21/quickref/#captions-prerecorded) ([Memahami 1.2.2](/WAI/WCAG21/Understanding/captions-prerecorded))
+  * [Deskripsi Audio atau Media Alternatif (Prarekaman) 1.2.3](/WAI/WCAG21/quickref/#audio-description-or-media-alternative-prerecorded) ([Memahami 1.2.3](/WAI/WCAG21/Understanding/audio-description-or-media-alternative-prerecorded))
+* **Kisah Pengguna**
+  * [Menjelaskan bagaimana takarir membantu siswa penyandang disabilitas rungu](/people-use-web/user-stories/#onlinestudent)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Sediakan instruksi yang jelas
+
+Pastikan instruksi, panduan, dan pesan eror jelas, mudah dimengerti, dan hindari bahasa teknis yang tidak perlu. Jabarkan persyaratan dari input, seperti format tanggal.
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Contoh: Instruksi mengkomunikasikan informasi apa yang harus disediakan oleh pengguna" class="example" %}
+{:/}
+
+<div class="errors">
+  <figure>
+    <div>
+      <form action="#" method="post">
+        <p id="password-desc">Kata sandi harus terdiri dari minimal enam karakter dengan setidaknya satu angka (0-9).</p>
+        <div class="row">
+          <label for="password">Kata sandi</label>
+          <input aria-describedby="password-desc" type="password" id="password" name="password" value="">
+        </div>
+      </form>
+    </div>
+  </figure>
+</div>
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include box.html type="start" title="Contoh: Pesan eror menunjukkan apa masalahnya dan, sebisa mungkin, bagaimana cara memperbaikinya" class="example" %}
+{:/}
+
+<div class="errors">
+  <figure>
+    <div>
+      <ol class="fa-ul error-list">
+        <li id="error_email">{% include_cached icon.html name="warning" label="Eror" %} <a href="javascript:return false">Nama pengguna 'superbear' sudah ada yang menggunakan.</a></li>
```

Comment:
@frirasyidi If you think best to keep the English "superbear", please wrap it in `<span lang="en"></span>`

Reply:
@remibetin 'superbear' is a name so I should not wrap it in <span lang="en"></span>, no?